extern/Python/CPython_netstd20/Readme.txt

```diff
@@ -0,0 +1 @@
+These assemblies are built from https://git.autodesk.com/Dynamo/pythonnet targeting netstandard2.0
```

Comment:
I added new assemblies for netstd20  only because I could not get tests to run on the pythonnet targeting netstd20
Once we get pythonnet CICD to run properly, we can just keep the new netstd20 assmeblies (since they are compatible with net48 and net6.0)

Reply:
Did the test run properly for you locally? The embedded test part of CICD for PythoNet is unstable (when running in docker) and has been for a while.

Reply:
Also, we have more changes coming into python net soon so we should resolve this sooner than later.

Reply:
I could not get the tests to run locally. 
Something with the nunit version is causing issues.